web-common/src/features/dashboards/pivot/pivot-data-store.ts

```diff
@@ -46,8 +47,13 @@ import {
  */
 function getPivotConfig(ctx: StateManagers): Readable<PivotDataStoreConfig> {
   return derived(
-    [useMetricsView(ctx), ctx.dashboardStore, useTimeControlStore(ctx)],
-    ([metricsView, dashboardStore, timeControls]) => {
+    [
+      useMetricsView(ctx),
+      ctx.dashboardStore,
+      useTimeControlStore(ctx),
+      measureFilterResolutionsStore(ctx),
```

Comment:
Since measure filter have no change in the pivot flow do you think we can just directly use it in `createPivotAggregationRowQuery` for all queries that call it?

Reply:
It feels like this is the perfect place. Since all settings from dashboard needed for querying that is not changed is passed through here it fits pretty well. It also avoids adding derived and set to get the async data from this piece.